digicampipe/utils/pulse_template.py

```diff
@@ -78,9 +100,8 @@ def plot(self, axes=None, **kwargs):
         t = np.linspace(self.time.min(), self.time.max(),
                         num=len(self.time) * 100)
 
-        axes.plot(self.time, self.amplitude, label='Template data-points',
-                  **kwargs)
-        axes.plot(t, self(t), label='Interpolated template')
+        axes.errorbar(self.time, self.amplitude, self.amplitude_std,
```

Comment:
does it work if self.amplitude_std is None ?

Reply:
![old_template_plot](https://user-images.githubusercontent.com/32672677/47435003-00090680-d7a4-11e8-8ec0-6f244f4beb5d.png)
I just tried with the old template file (without std) and it worked as it is (no errorbar shown)